.github/workflows/changeset.yml

```diff
@@ -162,3 +162,43 @@ jobs:
         run: |
           echo "Please include at least one tag in the core changeset file"
           exit 1
+
+  contracts-changeset:
+    name: Contracts Changeset Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4.2.1
+        with:
+          fetch-depth: 0
+
+      - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: files-changed
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
2x checking that we need to pass a token here.

Reply:
Removed in dfcc6c4b17f4cbbc76881d03f191c536ef15937a